  mic_name: "ReSpeaker 4 Mic Array"
  # mic_name: "sof-hda-dsp: - (hw:0,7)"
  # mic_name: "HD Pro Webcam C920"
  main_channel: 0
  sample_rate: 16000
  update_rate: 4.0
  transcribe_servers: {'0': 'https://ai.sweetie.bot/stt/', '1':'http://localhost:8577/'}
  key_combination: [ 'ctrl', 'w' ]
  call_llm: False
  sound_intensity_threshold: 300
  intensity_normalization_divider: 2000
  doa:
      #algorithm: 'respeaker'
      algorithm: 'NormMUSIC'
      #algorithm: 'SRP'
      #algorithm: 'FRIDA'
      #algorithm: 'MUSIC'
      #algorithm: 'TOPS'
      mic_coords: [ 0.02285, -0.02285, 0.02285, 0.02285, -0.02285, 0.02285, -0.02285, -0.02285 ]
      mic_channels: [ 1, 2, 3, 4 ]
      azimuth: [ -180.0, 180.0, 32 ]
      nfft: 1024
